archiver_app.py

```diff
@@ -84,15 +86,35 @@ def archive():
     if is_direct_archiving:
         try:
             direct_archiver = direct_archiver_from_config()
-            response = direct_archiver.archive_submission(submission_uuid)
+            archives_response = direct_archiver.archive_submission(submission_uuid)
             logger.info('Archiving finished with UUID=%s', submission_uuid)
-        except RestErrorException as rest_error:
+        except Exception as rest_error:
```

Comment:
This error handling section is very long.
Have you considered a function for each case?
Have you considered using an `except` clause for each exception type?
Have you considered using a dataclass or namedtumple for the error response object?

Reply:
Cleaned up